src/utils/_variables.scss

```diff
@@ -0,0 +1,16 @@
+$day-size: 100px;
+$day-count: 31;
+$border-size : $day-size / 100;
+$gap-size: $day-size / 100;
```

Comment:
why does border-size depend on day-size?

Reply:
There was a dicision to link a border dimension to the cell size  in order to improve appearance of cells of any size. 
Idea was to automate all was it is possible by use of singlew parameter - cell size. 
Your approach is clear , agree with one . Direct setting in that case is more obvious and logical.